Tools/Scripts/libraries/webkitscmpy/webkitscmpy/remote/git_hub.py

```diff
@@ -694,3 +697,14 @@ def files_changed(self, argument=None):
             for file in self.request('commits/{}'.format(commit.hash)).get('files', [])
             if file.get('filename')
         ]
+
+    def releases(self, name, tag_name, target_commitish='main', body='', draft=False, prerelease=False):
```

Comment:
Does this create or list releases? `releases` would indicate it just lists releases, but this looks like it _creates_ them? Should probably name `create_release` or something to that effect.